test/libsolidity/syntaxTests/natspec/false_natspec.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >0.4.24;
+
+contract C {
+    struct S {
+        uint x;
+    }
+
+    /// a - natspec comment ///
+    S a;
+    /** b - natspec comment **/
+    S b;
+    /// c - natspec comment, first line
+    /// second line
+    S c;
+    ///  d - natspec comment, single line
+    //// not a natspec line
+    S d;
+    //// e - not a natspec comment ////
+    S e;
+    /*** f - not a natspec comment ***/
+    S f;
+    /***/
+    S g;
+    /****/
+    S h;
+}
+// ----
+// Warning: (117-144): Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
+// Warning: (158-185): Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
+// Warning: (199-254): Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
+// Warning: (268-305): Documentation tag on non-public state variables will be disallowed in 0.7.0. You will need to use the @dev tag explicitly.
```

Comment:
Do you think adding a test in `test/libsolidity/SolidityNatspecJSON.cpp` makes sense?

Reply:
I agree, this is very hard to review like that.